index.html

```diff
@@ -116,14 +116,12 @@ <h1 class="text-center">About Us</h1>
                     <h2 class="text-white font-weight-bold py-2">Contact Us</h2>
                 </div>
                 <div class="col-lg-6 col-md-6 col-sm-12 col-12">
-                    <form action="">
+                    <form action="mailto:algophantom0101@gmail.com">
                         <div class="input-group">
                             <input type="text" class="form-control py-3" placeholder="Enter your Email">
                             <div class="input-group-append">
-                                <div class="input-group-text subscribe-button text-white px-5 font-weight-bold">
-                                    Contact Us
-                                </div>
-                            </div>
+                                <input class="btn btn-dark" type="submit" value="Contact Us">
```

Comment:
It is better to use the initial color only instead of using the bootstrap dark button as dark button on a dark background is not looking so good.